styles/modules/modals/_editListing.scss

```diff
@@ -261,13 +261,17 @@
             border-style: solid;
             padding: $pad;
             word-break: break-word;
-            min-width: 90px;
+           // min-width: 90px;
```

Comment:
Can we please keep the min-width? It allows the table to get a horizontal scrollbar when it has a lot of content. Otherwise things get very squished:

![image](https://cloud.githubusercontent.com/assets/794517/24157142/c6ed3794-0e15-11e7-8b7b-09e90abd05d7.png)